documenation/documentation.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Xenon- UI</title>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+
+  <link rel="stylesheet" href="../default.css" />
+  <link rel="stylesheet" href="./documentation.css">
+  <link rel="stylesheet" href="../Components/Lists/list.css">
+  <link rel="stylesheet" href="../Components//Avatar/avatar.css" />
+  <link rel="stylesheet" href="../Components/Alerts/alerts.css">
```

Comment:
Suggestion:
rather than importing all css you can create 1 main.css file and import these files there and import main file in HTML. its up to you

Reply:
Yes.